sakura_core/window/CEditWnd.cpp

```diff
@@ -4860,6 +4860,19 @@ ECharWidthCacheMode CEditWnd::GetLogfontCacheMode()
 	return CWM_CACHE_SHARE;
 }
 
+/*!
+	@brief 現在のズーム倍率を取得
+	@return 1.0を等倍とするズーム倍率
+*/
+double CEditWnd::GetFontZoom()
```

Comment:
このPRに含めなくてよいですが...

```suggestion
double CEditWnd::GetFontZoom() const noexcept
```

1. メンバーを変更しないので const にできます。
2. この関数の呼出し元で例外を考慮しなくてよいことを noexcept で示せます。